lib/resources/example_json/dstu2_examples_slot.rb

```diff
@@ -2,80 +2,247 @@ module Cerner
+              "lastUpdated": "2015-06-10T10:40:33.000-05:00"
+            },
+            "text": {
+              "status": "generated",
+              "div": "&lt;div&gt;&lt;p&gt;&lt;b&gt;Slot&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Type&lt;/b&gt;: Same Day&lt;/p&gt;&lt;p&gt;&lt;b&gt;Start&lt;/b&gt;: 2016-01-01T08:10:00.000-06:00&lt;/p&gt;&lt;p&gt;&lt;b&gt;End&lt;/b&gt;: 2016-01-01T09:10:00.000-06:00&lt;/p&gt;&lt;/div&gt;"
+            },
+            "extension": [
+              {
+                "url": "https://fhir-ehr.cerner.com/dstu2/StructureDefinition/scheduling-location",
+                "valueReference": {
+                  "reference": "Location/633867"
+                }
+              }
+            ],
+            "type": {
+              "coding": [
+                {
+                  "system": "http://snomed.info/sct",
+                  "code": "394581000",
+                  "display": "Community medicine"
+                }
+              ],
+              "text": "Same Day"
+            },
+            "schedule": {
+              "reference": "Schedule/21265426-633867-3121665-10"
+            },
+            "start": "2016-01-01T08:10:00.000-06:00",
+            "end": "2016-01-01T09:10:00.000-06:00"
+          }
         },
-        "schedule": {
-            "reference": "Schedule/166627791-4048128-9331486-0"
+        {
+          "fullUrl": "https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Slot/21265426-633867-3121665-15",
+          "resource": {
+            "resourceType": "Slot",
+            "id": "21265426-633867-3121665-15",
+            "meta": {
+              "versionId": "0",
+              "lastUpdated": "2015-06-10T10:40:33.000-05:00"
+            },
+            "text": {
+              "status": "generated",
+              "div": "&lt;div&gt;&lt;p&gt;&lt;b&gt;Slot&lt;/b&gt;&lt;/p&gt;&lt;p&gt;&lt;b&gt;Type&lt;/b&gt;: Same Day&lt;/p&gt;&lt;p&gt;&lt;b&gt;Start&lt;/b&gt;: 2016-01-01T08:15:00.000-06:00&lt;/p&gt;&lt;p&gt;&lt;b&gt;End&lt;/b&gt;: 2016-01-01T09:15:00.000-06:00&lt;/p&gt;&lt;/div&gt;"
+            },
+            "extension": [
+              {
+                "url": "https://fhir-ehr.cerner.com/dstu2/StructureDefinition/scheduling-location",
+                "valueReference": {
+                  "reference": "Location/633867"
+                }
+              }
+            ],
+            "type": {
+              "coding": [
+                {
+                  "system": "http://snomed.info/sct",
+                  "code": "394581000",
+                  "display": "Community medicine"
+                }
+              ],
+              "text": "Same Day"
+            },
+            "schedule": {
+              "reference": "Schedule/21265426-633867-3121665-15"
+            },
+            "start": "2016-01-01T08:15:00.000-06:00",
+            "end": "2016-01-01T09:15:00.000-06:00"
+          }
         },
-        "freeBusyType": "busy",
```

Comment:
If the lack of freeBusyType is a DAAKOS issue, can you add it back into the examples so they are more normative?

Reply:
Restored with https://github.com/cerner/fhir.cerner.com/pull/170/commits/14a8b9797d173634ac02067d8219abea5946619f